src/boot_O2/boot_0x80086FA0.c

```diff
@@ -0,0 +1,75 @@
+#include <ultra64.h>
+#include <global.h>
+
+/**
+ * Gets the next integer in the sequence of pseudo-random numbers.
+ */
+u32 Rand_Next(void) {
+    return sRandInt = (sRandInt * 1664525) + 1013904223;
```

Comment:
I think it would be beneficial to have these two magic numbers as `#define` at the top of this file. I suggest
```
#define RAND_MULTIPLIER 1664525
#define RAND_INCREMENT 1013904223
```